src/main/kotlin/org/pkl/lsp/ast/PklNode.kt

```diff
@@ -824,6 +824,8 @@ fun TreeSitterNode.toNode(project: Project, parent: PklNode?): PklNode? {
       when (children[1].type) {
         "**" -> PklExponentiationExprImpl(project, parent!!, this)
         "??" -> PklNullCoalesceExprImpl(project, parent!!, this)
+        "lineComment",
+        "ERROR" -> PklErrorImpl(project, parent!!, this)
```

Comment:
That's definitely a bug! But there are some problems with your solution:
- `lineComment` shouldn't become an error, otherwise the LSP will report a syntax error on the comment
- `blockComment` will still thrown an exception in cases like `foo /* asd */ || bar`

The problem here is the `when (children[1].type)` which assumes the first child is the operator, but that's not always true. We need better heuristics.

Reply:
Addressed handling of `blockComment`